Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppDeploy/DeployIISWebApp.ps1

```diff
@@ -70,7 +72,8 @@ function Run-RemoteDeployment
         [string]$adminPassword,
         [string]$winrmProtocol,
         [string]$testCertificate,
-        [string]$deployInParallel
+        [string]$deployInParallel,
+        [string]$webDeployPackage
```

Comment:
But this is not used anywhere in this function - except for the log messages. Should it be passed to Invoke-RemoteDeployment too?